src/main/java/se/sundsvall/supportmanagement/service/scheduler/emailreader/EmailReaderWorker.java

```diff
@@ -119,7 +119,15 @@ private boolean isErrandInactive(final ErrandEntity errand, final EmailWorkerCon
 		return config.getInactiveStatus() != null
 			&& errand.getStatus().equals(config.getInactiveStatus())
 			&& config.getDaysOfInactivityBeforeReject() != null
-			&& errand.getTouched().isBefore(OffsetDateTime.now().minusDays(config.getDaysOfInactivityBeforeReject()));
+			// errand.getTouched() will be null if session is not flushed.
+			&& calculateTouched(errand).isBefore(OffsetDateTime.now().minusDays(config.getDaysOfInactivityBeforeReject()));
+	}
+
+	private OffsetDateTime calculateTouched(ErrandEntity errand) {
```

Comment:
Is this necessary? Isn't this already perfomed in persistence-layer (see touched attribute in entity-class)

Reply:
If so: 

Why not remove the formula stuff (it doesn't seem to be portable anyway) and add this "return greatest date"-logic in the toched-attributes getter?

This seems to be a better place for this calculation i.m.h.o. (behaviour will be consistent throughout the service)

(Don't forget to remove the `@Formula` and add `@Transient`)

Reply:
Ok. But do we need the member touched (annotated with transient) ? Cant we just have a getTouched method in entity and that's it?

Reply:
Absolutely!